app/client/src/features/landing/CallToAction/CallToAction.tsx

```diff
@@ -50,28 +50,39 @@ export function CallToAction() {
             <Grid item xs={12} md={6} className={classes.header}>
                 <Image
                     alt='Prytaneum Logo'
-                    src='/static/prytaneum_logo2.svg' 
+                    src='/static/prytaneum_logo2.svg'
                     width={3483}
                     height={665}
                     objectFit='contain'
                 />
                 <Typography variant='h5' className={classes.subtitle}>
                     A crucial tool for a better democracy.
                 </Typography>
-                {user ? 
-                    <Button variant='contained' color='secondary' className={classes.button} onClick={() => router.push('/app/home')}>
+                {user ? (
+                    <Button
+                        variant='contained'
+                        color='secondary'
+                        className={classes.button}
+                        onClick={() => router.push('/app/home')}
+                    >
```

Comment:
Formatting looks weird here

Reply:
It was also a consequence of adding the data-test-id field. I ran prettier on it though so it should be the correct formatting.

Reply:
Maybe it was github printing it weird.  It looks normal now.

Reply:
Ah, yeah it was I see what happened now.  ignore the formatting comments